cric-predict/src/Components/Form.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
I don't agree with this naming convention component name should represent the business object it's supposed to render .. these shouldn't be named Forms either .. maybe a better name for this would be `AuthenticationComponent` and file name `Authentication.js`